docs/source/sysadmin_guide/configure_sgx.rst

```diff
@@ -0,0 +1,458 @@
+    $ sudo add-apt-repository 'deb http://repo.sawtooth.me/ubuntu/0.8/stable xenial universe'
+    $ sudo apt-get update
+    $ sudo apt-get install -y -q \
+      sawtooth \
+      python3-sawtooth-poet-sgx
+
+
+Configure the validator to use SGX PoET
+---------------------------------------
+
+After installing Sawtooth, add config settings so PoET will work properly.
+Create the file ``/etc/sawtooth/poet_enclave_sgx.toml``
+with your favorite editor (such as vi):
+
+.. code-block:: console
+
+    $ sudo vi /etc/sawtooth/poet_enclave_sgx.toml
+
+Add the following lines:
+
+.. code-block:: console
+
+    # spid is a 32-digit hex string tied to the enclave implementation
+
+    spid = 'B785C58B77152CBE7FD55EE3851C4990'
+
+    # ias_url is the URL of the Intel Attestation Service (IAS) server.
+
+    ias_url = 'https://test-as.sgx.trustedservices.intel.com:443'
+
+    # spid_cert_file is the full path to the PEM-encoded certificate file that was
+    # submitted to Intel in order to obtain a SPID
+
+    spid_cert_file = '/etc/sawtooth/maiden-lane-poet-linkable-quotes.pem'
+
+Next, copy the .pem certificate file you downloaded earlier.
+Replace [example] in the path below with the path to the certificate file on
+your local system:
+
+.. code-block:: console
+
+    $ sudo cp /[example]/maiden-lane-poet-linkable-quotes.pem /etc/sawtooth/maiden-lane-poet-linkable-quotes.pem
+
+Become the sawtooth user and change to ``/tmp``:
+
+.. code-block:: console
+
+    $ sudo -u sawtooth -s
+    $ cd /tmp
+
+Create validator keys:
+
+.. code-block:: console
+
+    $ sawtooth admin keygen
+
+.. note:: If you're configuring multiple validators, the steps below 
+    do not need to be repeated on subsequent validators. There is some
+    additional configuration required though. See 
+    :ref:`val-config` below.
+
+Create a genesis batch:
+
+.. code-block:: console
+
+    $ sawtooth config genesis --key /etc/sawtooth/keys/validator.priv -o config-genesis.batch
+
+Create and submit a proposal:
+
+.. code-block:: console
+
+    $ sawtooth config proposal create -k /etc/sawtooth/keys/validator.priv \
+    sawtooth.consensus.algorithm=poet \
+    sawtooth.poet.report_public_key_pem="$(cat /etc/sawtooth/ias_rk_pub.pem)" \
+    sawtooth.poet.valid_enclave_measurements=$(poet enclave --enclave-module sgx measurement) \
+    sawtooth.poet.valid_enclave_basenames=$(poet enclave --enclave-module sgx basename) \
+    sawtooth.poet.enclave_module_name=poet_enclave_sgx.poet_enclave \
+    -o config.batch
+
+You should see a couple lines of output showing that the SGX enclave has been initialized:
+
+.. code-block:: console
+
+    [12:03:58 WARNING poet_enclave] SGX PoET enclave initialized.
+    [12:03:59 WARNING poet_enclave] SGX PoET enclave initialized.
+
+There’s quite a bit going on in the previous command, so let’s take a closer look at what it accomplishes:
+
+1.  ``sawtooth.consensus.algorithm=poet``
```

Comment:
How about changing the numbered list to a "what"-style definition list?  I'd recommend putting each command arg on its own line (just remove the number), followed by the description (indented two spaces).  For example: 

      sawtooth.consensus.algorithm=poet
          Adds the public key that the validator registry transaction processor uses
          to verify attestation reports.

Also, I have a handful of wording and punctuation changes for these items. I've attached a text file with the formatting and writer changes. 

[proposal-cmd-rewrite.txt](https://github.com/hyperledger/sawtooth-core/files/1334453/proposal-cmd-rewrite.txt)